pkgs/development/libraries/openexrid/default.nix

```diff
@@ -0,0 +1,50 @@
+{ stdenv, fetchFromGitHub, unzip, re2, openfx, zlib, ilmbase, mesa, openexr }:
+
+stdenv.mkDerivation rec
+{
+  name = "openexrid-${version}";
```

Comment:
Per the [package naming guidelines](https://nixos.org/nixpkgs/manual/#sec-package-naming) I believe the name here should be `openexrid-unstable-${version}`.

Reply:
Corrected. Thank you.